transport/api/src/constants.rs

```diff
@@ -17,3 +20,12 @@ pub(crate) const RESERVED_SESSION_TAG_UPPER_LIMIT: u16 = 1024;
 /// The reserved tags are from range <0,[`RESERVED_SUBPROTOCOL_TAG_UPPER_LIMIT`]) and are
 /// specifically dedicated for the internal use by the subprotocols.
 pub(crate) const RESERVED_SUBPROTOCOL_TAG_UPPER_LIMIT: u16 = 16;
+
+/// Time within Start protocol must finish session initiation.
+pub(crate) const SESSION_INITIATION_TIMEOUT: Duration = Duration::from_secs(90);
```

Comment:
Also I would reduce this.

Reply:
Libp2p has timeout one way set to 30 seconds, so we can se this to 60 which is theoretical maximum.